board: add support for nrf54ls05a (#844)
nrfconnect/sdk-nrf-bm: doc/nrf-bm/release_notes/release_notes_changelog.rst

Boards
======

> * Added the a device nRF54LS05A to be supported by the existing ``bm_nrf54l05dk`` board target.

Reviewer: ```suggestion
* Added the nRF54LS05A SoC as an emulated variant of the existing ``bm_nrf54l05dk`` board target.
```
